charts/argo-server/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,58 @@
+---
+{{- $saName := printf "%s-account" .Release.Name }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ $saName }}
+{{- if .Values.rbac.create }}
+{{- $pspName := printf "%s-pod-running-policy" .Release.Name }}
+{{- $roleName := printf "%s-role" .Release.Name }}
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ $pspName }}
+spec:
+  privileged: false
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  fsGroup:
+    rule: RunAsAny
+  volumes:
+    - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ $roleName }}
+rules:
+  - apiGroups: ['policy']
+    resources: ['podsecuritypolicies']
+    verbs: ['use']
+    resourceNames: ['{{ $pspName }}']
+  - apiGroups: ['']
+    resources: ['configmaps']
```

Comment:
Quick question - I've seen the above format as well as
`    resources:
    - configmaps`
as acceptable syntax. Is one preferred over the other, or are the brackets used when we know there will be additional items added?

Reply:
Both are acceptable ways to specify an array, I don't have a clear preference for one over another. I generally gravitate towards JSON-ish syntax like this when there's only one element, and flip to the YAML-style list syntax with `-`s when there's more than one.